tilequeue/query/postgres.py

```diff
@@ -67,9 +64,15 @@ def __init__(self, sources, query_generator):
     def __call__(self, zoom, bounds):
         queries = []
         for source in self.sources:
-            if source.start_zoom <= zoom:
-                query = self.query_generator(source.template, bounds, zoom)
-                queries.append(query)
+            template_queries = []
+            for template_spec in source.template_specs:
+                if template_spec.zoom_start <= zoom < template_spec.zoom_stop:
```

Comment:
I think it should be inclusive. IIRC, we're inclusive about zoom ranges in the post-processing functions, [e.g](https://github.com/tilezen/vector-datasource/blob/4eb1fbd1cab6194713761470b2fdb57eb43d3c01/vectordatasource/transform.py#L2072-L2076).

On second thoughts, I don't mind whether it's inclusive or exclusive, as long as we're consistent. I think the one-past-the-end counting is easier for us to reason about internally, but perhaps the inclusive ranges are more intuitive for non-developers?